### DISCUSSION TITLE
Help with Grid [Bootstrap 5]

### DISCUSSION THREAD

**Discussion by minecraftchest1** [twbs/bootstrap#35180] [Q&A]
Hello. I have a webpage that I am working on ( a browser-based remote for Roku devices). This is my first bootstrap project. The problem I have is with the bootstrap grid system.
When I create a grid, the controls appear to get scattered all over the page. I have tried to size the container down to a size that is reasonable for my project, but it does not appear to make a difference. Webpage source can be found https://gitlab.com/-/snippets/2188019.
I have attached what the page looks like, as well as what I expected it to look like (using divs and tables for layout).
![image](https://user-images.githubusercontent.com/42992675/136965470-cda7bfd2-41fd-4ab0-ad49-74c89d6a060e.png)
![image](https://user-images.githubusercontent.com/42992675/136965515-86e94e56-b39d-4378-b257-c8e44dfe7ba8.png)

**Comment by adityadees** [comment]
what result do you expect?

**Reply by minecraftchest1** [reply]
The second image is the result I expected to receive.
